public/index.html

```diff
@@ -38,6 +38,7 @@
 
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/FortAwesome/Font-Awesome@5.13.0/css/all.min.css">
 
+    <script src="https://unpkg.com/@codersrank/activity@0.9.14/codersrank-activity.min.js"></script>
```

Comment:
We should download this file, move it into a `utils` folder and import it from there so it can get optimized during compile time.

Reply:
well it's in npm https://www.npmjs.com/package/@codersrank/activity I think that should be used.

Reply:
Yes, It is in npm. I'll update this.